src/backend/columnar/sql/udfs/cstore_tableam_handler/10.0-1.sql

```diff
@@ -0,0 +1,10 @@
+CREATE OR REPLACE FUNCTION cstore.cstore_tableam_handler(internal)
+    RETURNS table_am_handler
+    LANGUAGE C
+AS 'MODULE_PATHNAME', 'cstore_tableam_handler';
+
+COMMENT ON FUNCTION cstore.cstore_tableam_handler(internal)
+    IS 'internal function returning the handler for cstore tables';
+
+-- TODO add check to only create if not already existing
+CREATE ACCESS METHOD cstore_tableam TYPE TABLE HANDLER cstore.cstore_tableam_handler;
```

Comment:
This needs to be done before checkin, right?

Reply:
The TODO is outdated. The check is performed in the `citus_pg_finish_upgrade` snippet. Will remove the TODO.